build/webpack.karma.config.js

```diff
@@ -1,19 +1,28 @@
 require('babel-polyfill');
 
+const { argv } = process;
+const isDebug = argv.find((arg) => {
+    return arg === '--auto-watch' || arg === '--no-single-run';
+});
+
 const { IgnorePlugin } = require('webpack');
 const commonConfig = require('./webpack.common.config');
 
 const baseConfig = commonConfig('en-US');
 
 const config = Object.assign(baseConfig, {
-    devtool: 'inline-source-map',
     resolve: {
         alias: {
             sinon: 'sinon/pkg/sinon'
         }
     }
 });
 
+if (isDebug) {
+    config.devtool = 'inline-source-map';
```

Comment:
Will change this to the other source map type once I check it on all of our browsers

Reply:
changed